openwisp_controller/pki/api/urls.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
+from django.urls import path
+
+from . import views as api_views
+
+app_name = 'openwisp_controller'
+
+
+def get_pki_api_urls(api_views):
+    """
+    returns:: all the API urls of the PKI app
+    """
+    if getattr(settings, 'OPENWISP_CONTROLLER_PKI_API', True):
+        return [
+            path('ca/', api_views.ca_list, name='ca_list'),
+            path('ca/<str:pk>/', api_views.ca_detail, name='ca_detail'),
+            path('ca/<str:pk>/crl', api_views.crl_download, name='crl_download'),
+            path('cert/', api_views.cert_list, name='cert_list'),
+            path('cert/<str:pk>/', api_views.cert_detail, name='cert_detail'),
```

Comment:
| SN | /api/v1/pki/cert/str:pk                       | Status | Details |
|----|-----------------------------------------------|:------:|:-------:|
| 1  | [SU][O1A][O1O] Can view only appropriate Cert |  Pass  |         |
| 2  | [SU][O1A][O1O] Can edit only appropriate Cert |  Pass  |         |
| 3  | [SU][O1A] Correct data is shown               |  Pass  |         |
| 4  | [SU][01A] Can delete only appropriate data    |  Pass  |         |